index.html

```diff
@@ -8,6 +8,10 @@
         <em>No upcoming events... <i class="fa fa-frown-o"></i></em>
     </div>
 </div>
+<h2 class="visible-xs"><i class="fa fa-map-marker"></i> Location</h2>
+<div class="visible-xs">
```

Comment:
Hi @AverageMarcus, shouldn't this `h2` live inside the `div` as they're related? That would mean there only needs to be one `visible-xs` switch too.